activemq-unit-tests/src/test/java/org/apache/activemq/network/NetworkConnectionsTest.java

```diff
@@ -5,9 +5,9 @@
  * The ASF licenses this file to You under the Apache License, Version 2.0
  * (the "License"); you may not use this file except in compliance with
  * the License.  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
```

Comment:
Seems like this should be unwound...but if a change actually needs made, I'd make the licence header a comment rather than Javadoc.